azure-pipelines.yml

```diff
@@ -501,7 +502,7 @@ jobs:
   - bash: |
       . util/build_consts.sh
 
-      util/make_distribution.sh
```

Comment:
did you also mean to remove this old file? (i.e., did you just want to move the location of this script?)

Reply:
I left it intentionally to remove in a followup pr that's mentioned here. It will avoid CI failures this way.